app/src/main/res/layout/activity_main.xml

```diff
@@ -54,6 +54,12 @@
             <include layout="@layout/content_main_keytypes" />
         </ViewFlipper>
 
+        <include
+            android:id="@+id/no_users_screen"
+            layout="@layout/content_main_no_users"
+            android:visibility="gone"/>
```

Comment:
Haha sorry, the include is used because the 'framework' (toolbar and buttons) stays the same and the content changes.